src/libs/SearchQueryUtils.ts

```diff
@@ -162,51 +161,6 @@ function getFilters(queryJSON: SearchQueryJSON) {
     return filters;
 }
 
-/**
- * Given a filter name and its value, this function returns the corresponding ID found in Onyx data.
- * Returns a function that can be used as a computeNodeValue callback for traversing the filters tree
- */
-function getFindIDFromDisplayValue(cardList: OnyxTypes.CardList, taxRates: Record<string, string[]>) {
```

Comment:
this is no longer needed because now `substitutionsMap` always keeps the connection between `id` and name